app/forms/candidate_interface/other_qualification_type_form.rb

```diff
@@ -23,9 +23,10 @@ class OtherQualificationTypeForm
     attr_accessor :qualification_type
     attr_accessor :other_uk_qualification_type
     attr_accessor :non_uk_qualification_type
+    attr_accessor :no_other_qualification
 
-    validates :qualification_type, presence: true
-    validates :qualification_type, inclusion: { in: ALL_VALID_TYPES, allow_blank: false }
+    validates :qualification_type, presence: true, unless: :no_other_qualification
+    validates :qualification_type, inclusion: { in: ALL_VALID_TYPES, allow_blank: false }, unless: :no_other_qualification
```

Comment:
I think it's worth adding a few tests into the validations for this new behaviour